packages/vertica-nodejs/lib/connection-parameters.js

```diff
@@ -90,6 +89,18 @@ class ConnectionParameters {
       writable: true,
       value: val('password', config),
     })
+    Object.defineProperty(this, 'oauth_access_token', {
+      configurable: true,
+      enumerable: false,
+      writable: true,
+      value: val('oauth_access_token', config),
+    })
+
+    // user is required for non-OAuth connections
+    this.user = val('user', config)
+    if (!this.user && !this.oauth_access_token) {
+      this.user = process.platform === 'win32' ? process.env.USERNAME : process.env.USER
```

Comment:
Can you add a comment here so that we will log a notice once we have that option. Something like 
`// TODO: log a notice to the user that the user property was taken from the environment once we fully support logging`